client/app/cloud/project/compute/loadbalancer/cloud-project-compute-loadbalancer.service.js

```diff
@@ -0,0 +1,103 @@
+class CloudProjectComputeLoadbalancerService {
+    constructor ($q, OvhApiCloudProjectIplb, OvhApiIpLoadBalancing) {
+        this.$q = $q;
+        this.OvhApiCloudProjectIplb = OvhApiCloudProjectIplb;
+        this.OvhApiIpLoadBalancing = OvhApiIpLoadBalancing;
+    }
+
+    getLoadbalancer (id) {
+        // Get loadbalancer
+        return this.OvhApiIpLoadBalancing.Lexi().get({
+            serviceName: id
+        }).$promise.then(loadbalancer => {
+            if (loadbalancer.state !== "ok") {
+                return loadbalancer;
+            }
+            // Find the frontend http 80 if exists
```

Comment:
This I can understand myself.  However, some explanations as to why we're looking for fhat port 80 http frontend would be nice.